src/BizHawk.Client.EmuHawk/AVOut/FFmpegDownloaderForm.cs

```diff
@@ -20,7 +20,7 @@ public FFmpegDownloaderForm()
 			txtLocation.Text = FFmpegService.FFmpegPath;
 			txtUrl.Text = FFmpegService.Url;
 
-			if (OSTailoredCode.IsUnixHost) textBox1.Text = string.Join("\n", textBox1.Text.Split('\n').Take(3)) + $"\n\n(Linux user: Create a symlink with the below filename pointing to the ffmpeg binary with version {FFmpegService.Version}.)";
+			if (OSTailoredCode.IsUnixHost) textBox1.Text = string.Join("\n", textBox1.Text.Split('\n').Take(3)) + "\n\n(Linux user: If installing manually, you can use a symlink.)";

```

Comment:
I think we should also tell the user right away what version we consider compatible, so they know what to look for if they can't use our build.